drivers/integration/docker/docker.go

```diff
@@ -199,14 +200,61 @@ func (d *driver) Mount(
 		return "", nil, err
 	}
 
-	return d.volumeMountPath(mountPath), vol, nil
+	return d.subVolumeMountPath(mountPath), vol, nil
 }
 
 // Unmount will unmount the specified volume by volumeName or volumeID.
 func (d *driver) Unmount(
 	ctx types.Context,
 	volumeID, volumeName string,
 	opts types.Store) error {
+
+	log.WithFields(log.Fields{
```

Comment:
Hi @clintonskitson,

This should just be `ctx.WithFields...`. The context type is a `log.FieldLogger`.